magento/pwa-studio
# [doc] environment variables documentation

## Description

* Updates the build commands in the pwa-devdoc's project to fit the pattern used in the pwa-studio projects. 
* Creates a script to build auto-generated documentation content from the `/packages/pwa-buildpack/envVarDefinitions.json` file
* Creates a draft topic that includes the auto-generated documentation

## Related Issue

Closes #1563 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
2. Build the HTML: `npm run build`
3. Verify the auto-generated doc is created at: `src/_includes/auto-generated/buildpack/reference/envVarDefinitions.md`
4. Verify the markdown file contains the same data as the `/packages/pwa-buildpack/envVarDefinitions.json` file

**Note:**
_There is no need to run the HTML server since this topic will not be published until the actual buildpack CLI feature is released._

## Screenshots / Screen Captures (if appropriate)
n/a

## Proposed Labels for Change Type/Package




## Checklist:
